tests/test_person.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from classes.person import Person
+
+
+class PersonClassTest(unittest.TestCase):
+    def test_full_name_only_returns_strings(self):
+        with self.assertRaises(ValueError, msg='Only strings are allowed as names'):
+            my_class_instance = Person("staff", "Peter", 1234)
+            my_class_instance.full_name()
```

Comment:
This test is repetitive, since you are already testing that a name provided can only be a string. You could consider testing that the `full_name` returned actually contains the first name and last name separated by a space.